source/documentation/tools/create-a-derived-table/project-structure.md

```diff
@@ -1,30 +1,384 @@
+
+### Intermediate: Files and folders
+
+Let’s take a look at the intermediate layer of our project to understand the purpose of this stage more concretely.
+
+```shell
+models/intermediate
+└── courts
+    └── courts_intermediate
+        ├── courts_intermidiate_properties.yml
+        └── int_int_xhibit_and_common_platform_receipts_unioned.sql
+```
+
+- **Folders**
+  - ✅ **Subdirectories based on business groupings.** Much like the staging layer, we’ll house this layer of models inside their own `intermediate` database. Unlike the staging layer, here we shift towards being business-conformed, splitting our models up into subdirectories not by their source system, but by their area of business concern.
+- **File names**
+  - `✅ int_[entity]s_[verb]s.sql` - the variety of transformations that can happen inside of the intermediate layer makes it harder to dictate strictly how to name them. The best guiding principle is to think about _verbs_ (e.g. `pivoted`, `aggregated_to_user`, `joined`, `fanned_out_by_quantity`, `funnel_created`, etc.) in the intermediate layer. In our example project, we use an intermediate model to pivot payments out to the order grain, so we name our model `int_payments_pivoted_to_orders`. It’s easy for anybody to quickly understand what’s happening in that model, even if they don’t know [SQL](https://mode.com/sql-tutorial/). That clarity is worth the long file name. It’s important to note that we’ve dropped the double underscores at this layer. In moving towards business-conformed concepts, we no longer need to separate a system and an entity and simply reference the unified entity if possible. In cases where you need intermediate models to operate at the source system level (e.g. `int_shopify__orders_summed`, `int_core__orders_summed` which you would later union), you’d preserve the double underscores. Some people like to separate the entity and verbs with double underscores as well. That’s a matter of preference, but in our experience, there is often an intrinsic connection between entities and verbs in this layer that make that difficult to maintain.
+
+:::tip Don’t over-optimize too early!
+The example project is very simple for illustrative purposes. This level of division in our post-staging layers is probably unnecessary when dealing with these few models. Remember, our goal is a _single_ _source of truth._ We don’t want finance and marketing operating on separate `orders` models, we want to use our dbt project as a means to bring those definitions together! As such, don’t split and optimize too early. If you have less than 10 marts models and aren’t having problems developing and using them, feel free to forego subdirectories completely (except in the staging layer, where you should always implement them as you add new source systems to your project) until the project has grown to really need them. Using dbt is always about bringing simplicity to complexity.
```

Comment:
Should it not be "expect in the staging layer where you should always use them if you add new source systems"

Reply:
What about replacing "We don’t want finance and marketing operating on separate `orders` models" with something like "If two teams of analysts both have questions about counts of criminal court cases, we want them to be drawing upon the same data table(s)"

Reply:
re first point: Whats the difference between the two sentences?
re second poin, I have added the suggested changes

Reply:
maybe you should ignore that first point in hindsight. although I think having a staging domain will change the guidance.